lib/tasks/delete_routes.rake

```diff
@@ -0,0 +1,27 @@
+desc "Delete Routes Ids"
+task :delete_routes_ids => :environment do |t, args|
```

Comment:
These aren't route ids. They're `HtmlAttachment` ids from Whitehall. Might be worth naming them appropriately just so as it is clear what we are doing in this task.